charts/connect/values.yaml

```diff
@@ -73,10 +74,5 @@ operator:
     create: "{{ .Values.operator.create }}"
     name: onepassword-connect-operator
 
-service:
-  type: Nodeport
-  port: 8080
-
-acceptanceTests:
```

Comment:
The linter fails, I'm guessing this line got accidentally removed during your merge.

Reply:
Thanks for the catch. Should be restored.